spec/intl.html

```diff
@@ -1791,6 +1772,30 @@ <h1>
             In a _calendar_ such as *"japanese"* where eras do not start and end at year and/or month boundaries, note that the returned List should contain *"era"* and *"eraYear"* if _keys_ contains *"day"*, *"month"*, or *"monthCode"* (not only if it contains *"era"*, *"eraYear"*, or *"year"*, as in the example above) because it's possible for changing the day or month to cause a conflict with the era.
           </emu-note>
         </emu-clause>
+
+        <emu-clause id="sec-temporal-calendarresolvefields" type="abstract operation">
+          <h1>
+            CalendarResolveFields (
+              _calendar_: a String,
+              _fields_: an Object,
+              _type_: ~date~, ~year-month~, or ~month-day~,
+            ): either a normal completion containing ~unused~ or a throw completion
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>
+              It performs implementation-defined processing to validate that _fields_ (which describes a date in the built-in calendar identified by _calendar_) is sufficiently complete to satisfy _type_ and not internally inconsistent, and merges data that can be represented in multiple forms into standard properties of _fields_ (for example, merging *"month"* and *"monthCode"* into *"month"* and merging *"era"* and *"eraYear"* into *"year"*).
```

Comment:
> which describes a date

To me, "date" implies all YMD components, but can't the incoming object include only a part of a date, e.g. YM or MD? Is there another term that could be used here? "Date or part of a date" ?

> standard properties of _fields_

Is it worth defining "standard properties" precisely here?  

> merges data

Does this mean that it mutates `_fields_`?  Or something else?

Reply:
> > which describes a date
> 
> To me, "date" implies all YMD components, but can't the incoming object include only a part of a date, e.g. YM or MD? Is there another term that could be used here? "Date or part of a date" ?

Done.

> > standard properties of _fields_
> 
> Is it worth defining "standard properties" precisely here?

I don't think so, because it's calendar-specific and not observable by ECMAScript code (output from CalendarResolveFields is always consumed completely by CalendarDateToISO or CalendarMonthDayToISOReferenceDate, both of which are also implementation-defined).

> > merges data
> 
> Does this mean that it mutates `_fields_`? Or something else?

Yes, it mutates `_fields_`. I've update the text to hopefully clarify that.